subsys/fs/fcb/fcb_elem_info.c

```diff
@@ -63,25 +48,78 @@ fcb_elem_crc8(struct fcb *fcb, struct fcb_entry *loc, uint8_t *c8p)
 	return 0;
 }
 
-int fcb_elem_info(struct fcb *fcb, struct fcb_entry *loc)
+/* Given the offset in flash sector, calculate the FCB entry data offset and size, and calculate
+ * the expected endmarker.
+ */
+int
+fcb_elem_endmarker(struct fcb *_fcb, struct fcb_entry *loc, uint8_t *emp,
+		   bool fixed_endmarker_allow)
 {
+	uint8_t tmp_str[FCB_TMP_BUF_SZ];
+	int cnt;
+	uint16_t len;
 	int rc;
-	uint8_t crc8;
-	uint8_t fl_crc8;
+
+	(void)fixed_endmarker_allow;
```

Comment:
What does this do ?

Reply:
It is removing a unused variable compilation warning if `CONFIG_FCB_ALLOW_FIXED_ENDMARKER` is not enabled